frontend/src/assets/stylesheets/components/post-buttons.sass

```diff
@@ -21,6 +21,8 @@
     text-transform: uppercase
     width: 180px
 
+    transition: $transition-duration $transition-function
+
     &:disabled
```

Comment:
Cursor pointer when hovered over disabled button (hand with pointing finger) implies that this button might actually be clickable. Please change to something more suitable.

Reply:
Good point, there should be cursor not allowed from begining